Makefile

```diff
@@ -30,6 +30,8 @@ endif
 OS = $(shell go env GOOS)
 ARCH = $(shell go env GOARCH)
 
+OPERATOR_NAMESPACE ?=
```

Comment:
Why do we need `OPERATOR_NAMESPACE` when we already have `NAMESPACE`?

Reply:
removed.